MAUI/Autocomplete/UI-Customization.md

```diff
@@ -739,17 +739,27 @@ x:Name="autoComplete" >
 
 {% highlight c# %}
 
- autoComplete.DropdownOpening += AutoComplete_DropdownOpening;
+    SfAutoComplete autoComplete = new SfAutoComplete();
+    autoComplete.DropdownOpening += AutoComplete_DropdownOpening;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+{% tabs %}
 
-  private void AutoComplete_DropdownOpening(object sender, CancelEventArgs e)
+{% highlight c# %}
+    
+ private void AutoComplete_DropdownOpening(object sender, CancelEventArgs e)
  {
-     e.Cancel = true;
+     e.Cancel = true; // If you want to restrict the dropdown open then set the e.Cancel is true. 
```

Comment:
e.Cancel = true; // If you want to restrict the dropdown, open it, then set the e.Cancel to true.